### PR TITLE
Update cache example with redis

### PR DESCRIPTION
**Pull request by hskrasek**
This feature aims to add an example of using the redis adapter with the `CacheManager`.

### CODE REVIEW

File: components/cache/index.php [hunk 1]
```diff
@@ -23,10 +24,10 @@
     // which are loaded from the config class in the container.
     // More about the config class can be found in the config component; for now we will use an array
     $container['config'] = [
-        'cache.default' => 'file',
+        'cache.default'     => 'file',
         'cache.stores.file' => [
```
Comment (mattstauffer) [inline components/cache/index.php]: I don't like this indentation style; I removed it from your previous commit intentionally. :)

File: components/cache/index.php [hunk 2]
```diff
@@ -49,4 +50,37 @@
...
+            ],
+        ]
+    ];
+
+    $container['redis'] = new Database($container['config']['database.redis']);
+
+    $cacheManager = new CacheManager($container);
+
+    $cache = $cacheManager->store();
+    // $cache = $cacheManager->store('redis');
```
Comment (mattstauffer) [inline components/cache/index.php]: Could you explain in a code comment why this line is commented out? Thakns!

File: components/cache/index.php (hunk 1 above)
Comment (hskrasek) [inline components/cache/index.php]: My bad, auto formatter. Removing

File: components/cache/index.php (hunk 2 above)
Comment (hskrasek) [inline components/cache/index.php]: Sure thing, I removed it originally since its in the default example above, but I can just add it back in

**Comment by jeffreydevreede** [comment]
Got it working using these examples, thanks!
This should be merged :+1:

**Comment by mattstauffer** [comment]
@jeffreydevreede thanks for the heads up. I'll make a few tweaks and merge. Thanks Hunter for your work, sorry for slow merge!

**Comment by mattstauffer** [comment]
@hskrasek Thanks so much for this contribution!

**Pull request merged**